CHANGELOG.md

```diff
@@ -1,3 +1,250 @@
+- Fix Github Actions CI - run MSSQL as root again (#3456)
+- Fix `ejabberd_c2s` - don't drop the host type in `update_stanza` (#3464)
+- Adding `queryId` to archived inbox messages (#3482, #3492)
+- MAM - Avoid asking for user exist with a room (#3470)
+- Fix starting new caches (#3508, #3518)
+- Add `server_name_indication_host` config option (#3510)
+- Less MIM compiler warnings (#3524)
+- Add config flag to be able to set sni hostname matching (#3528)
+- Fix `is_configured` in async pools (#3566)
+- Do not call the obsolete backend API in CT hook (#3570)
+- Fix metrics for auth backend not being reported (#3585)
+- Fix `cowboy_static` options (#3618)
+- Fix `mod_ping` with `host_types` (#3638)
+- Use metric probes instead of eval for slow calculated metrics (#3644)
+- Fix flaky tests and speed up CI:
+   - Gather list of failed test cases in separate issue (#3361)
+   - Fix `rest_client_SUITE:muc:messages_can_be_paginated_in_room` (#3363)
+   - Fill the gaps in the event table for `service_domain_db` (#3365)
+   - Run muclight inbox tests one by one instead of parallel (#3374)
+   - Print preset in tools/circle-publish-github-summary-comment.sh (#3378)
+   - Enable `mod_last` only on its test suite (#3413)
+   - Enable DB modules in tests only when used (#3416)
+   - Retry insert MAM prefs on deadlock (#3421)
+   - Fix `domain_removal_SUITE:last_removal` (#3422)
+   - Remove wait from `unavailable_resources_dont_get_carbons` (#3431)
+   - Run tests in pubsub:basic in parallel (#3432)
+   - Use in-memory riak backend to speed-up IO (#3434)
+   - Fix `sm_SUITE` (#3436)
+   - Use `wait_for_archive_size` inside `send_rsm_messages` helper (#3443)
+   - Avoid a race condition in `sm_SUITE:resume_session_state_stop_c2s` (#3444)
+   - Fix `offline_SUITE:max_offline_messages_reached` (#3445)
+   - Reroute presences correctly on reconnect (SM fix) (#3459)
+   - Wait for async writers to flush messages in tests (#3466)
+   - Inbox flaky tests fixes (#3630)
+
+### Other
+
+- Faster, more stable CircleCI on docker with DB backends (#3401, #3519, #3530)
+- Updating dependencies and cleaning the rebar.lock file (#3425, #3430, #3465, #3490, #3520, #3531, #3533, #3573, #3657)
+- Document low-level MAM options (#3329)
+- Document  MUC Light cache affiliations (#3509)
+- Minor documentation improvements (#3369, #3400, #3404, #3412, #3438, #3532, #3545)
+- Update CONTRIBUTING.md (#3622)
+
+## Commits, merged PRs and closed issues
+- [List of merged PRs](https://github.com/esl/MongooseIM/pulls?q=is%3Apr+is%3Amerged+milestone%3A5.1.0)
+
+- [List of closed issues](https://github.com/esl/MongooseIM/issues?q=is%3Aissue+is%3Aclosed+closed%3A2021-10-08..2022-06-09)
+
+- [Repository history for this release](https://github.com/esl/MongooseIM/graphs/contributors?from=2021-10-08&to=2022-06-09&type=c)
+
+[List of merged PRs based on merge date]: # (https://github.com/esl/MongooseIM/pulls?q=is%3Apr+base%3Amaster+merged%3A%222021-10-08..2022-06-09%22+sort%3Acreated-asc+)
```

Comment:
This looks suspicious - it is not rendered in the Markdown view.

Reply:
I think it is a comment or smth like this. It was like this in the previous releases, and I copied it.